### PR TITLE
chore: test executor

### PR DESCRIPTION
**Pull request by dmi3y**
Testing executor setup and passing in $foo env variable as environment

bundle__14__-_dmi3y_circle-ci-test

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
+    docker:
+      - image: cimg/base:2021.04
+        environment:
+          FOO: $foo
+
+      - image: alpine
+        environment:
+          FOO: $foo
+        # Need foo env var here, but getting literal as $foo
+        command: ["sh", "-c", "env"]
```
Comment (dmi3y) [inline .circleci/config.yml:14]: No env passed inside the sub container

Comment (dmi3y) [reply .circleci/config.yml:14]: @buchs ^^

Comment (buchs) [reply .circleci/config.yml:14]: What this suggests to me is that the environment variable "foo" was not set in the Circle CI environment variables for the project.

Comment (dmi3y) [reply .circleci/config.yml:14]: It was, I believe, but good point let me doublecheck that!

Comment (dmi3y) [reply .circleci/config.yml:14]: Yes it is set up. I can see env variable in the host but not in sub container (alpine).
bundle__18__-_dmi3y_circle-ci-test

UPD: my bad I put both uppercase and lower cased env variables (`FOO` and `foo`) into the settings, still no luck.

Comment (dmi3y) [reply .circleci/config.yml:14]: Oh, and what I figured out it does not work the host as well, at least for build job...
Cursor_and_build__3__-_buchs_circle-ci-test

**Pull request merged**